src/components/views/dialogs/security/AccessSecretStorageDialog.tsx

```diff
@@ -343,6 +343,14 @@ export default class AccessSecretStorageDialog extends React.PureComponent<IProp
                 )}</p>
 
                 <form className="mx_AccessSecretStorageDialog_primaryContainer" onSubmit={this.onPassPhraseNext}>
+                    <input
+                            className="mx_AccessSecretStorageDialog_hidden-encryption-username-for-auto-fill-password"
+                            name="username" // make it a little easier for browser's remember-password
+                            key="username_input"
+                            type="text"
+                            placeholder={_t("Username").toLocaleLowerCase()}
```

Comment:
why a placeholder for an invisible field?

Reply:
Don't know exactly how this works with the auto fill on chrome but I can try without.